desktop/index.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Docker Desktop overview
+keywords: Desktop, Docker, GUI, run, docker, local, machine
+title: Docker Desktop overview
+toc_min: 1
+toc_max: 2
+---
+
+Docker Desktop is an easy-to-install application for your Mac or Windows environment that enables you to build and share containerized applications and microservices. Docker Desktop includes [Docker Engine](../engine/), Docker CLI client, [Docker Compose](../compose/), [Notary](../notary/getting_started/), [Kubernetes](https://github.com/kubernetes/kubernetes/), and [Credential Helper](https://github.com/docker/docker-credential-helpers/).
+
+Docker Desktop works with your choice of development tools and languages and gives you access to a vast library of certified images and templates in Docker Hub. This enables development teams to extend their environment to rapidly auto-build, continuously integrate and collaborate using a secure repository.
+
+Some of the key features of Docker Desktop include:
+
+* Ability to containerize and share any application on any cloud platform, in multiple languages and frameworks
+* Easy installation and setup of a complete Docker development environment
+* Includes the latest version of Kubernetes
+* Automatic updates with our preview channel - Desktop Edge and Stable versions
+* On Windows, the ability to toggle between Linux and Windows Server environments to build applications
+* Fast and reliable performance with native Windows Hyper-V virtualization
+* Ability to work natively on Linux through WSL 2 on Windows machines
+* Volume mounting for code and data, including file change notifications and easy access to running containers on the localhost network
+* In-container development and debugging with supported IDEs
+
+## Download and install
+
+Docker Desktop is available for Mac and Windows. For download information, system requirements, and installation instructions, see:
+
+* [Install Docker Desktop on Mac](../docker-for-mac/install.md)
+* [Install Docker Desktop on Windows](../docker-for-windows/install.md)
+
+## Get started
+
+For information on how to get to get started with Docker Desktop and to learn about various UI options and their usage see:
+
+* [Get started with Docker Desktop on Mac](../docker-for-mac/index.md)
+* [Get started with Docker Desktop on Windows](../docker-for-windows/index.md)
+
+## Stable and Edge versions
+
+Docker Desktop offers Stable and Edge download channels.
+
+The Stable release provides a general availability release-ready installer for a fully baked and tested, more reliable app. The Stable version of Docker Desktop includes the latest released version of Docker Engine. The release schedule is synced with Docker Engine releases and patch releases. You can choose to opt out of the usage statistics and telemetry data on the Stable channel.
```

Comment:
I would change this to:
every 3 months for major releases, with patch releases to fix minor issue or stay up to date with Docker Engine - as required